estate/models/est_estate_property.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError, UserError
+
+
+class EstateProperty(models.Model):
+    _name = "est.estates.property"
+    _description = "Estates Property"
+    _order = "id desc"
+
+    name = fields.Char('Title', required=True, default="Unknown")
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date('Available From', copy=False,
+                                    default=fields.Date.add(fields.Date.today(), months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False, compute='_compute_selling_price')
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer('Living Area (sqm)')
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer('Garden Area (sqm)')
+    garden_orientation = fields.Selection(
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')])
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        selection=[('new', 'New'), ('offer received', 'Offer Received'), ('offer accepted', 'Offer Accepted'),
+                   ('sold', 'Sold'), ('cancelled', 'Cancelled')],
+        default='new')
+    type_id = fields.Many2one("est.estates.property.type", string="Property Type")
+    salesman = fields.Many2one("res.users", default=lambda self: self.env.user)
+    buyer = fields.Many2one("res.partner", copy=False, readonly=True)
+    tag_ids = fields.Many2many("est.estates.property.tag", string="Tags")
+    offer_ids = fields.One2many("est.estates.property.offer", "property_id", string="Offers")
+    total_area = fields.Integer('Total Area (sqm)', compute='_set_area', readonly=True)
+    best_price = fields.Float(compute='_compute_best_price')
+
+    _sql_constraints = [
+        ('Check_expected_price', 'CHECK(expected_price > 0)',
+         'The expected price should be greater than 0!')
+    ]
+    _sql_constraints = [
+        ('Check_selling_price', 'CHECK(selling_price >= 0)',
+         'The selling price should be greater or equal to 0')
```

Comment:
good practice to put the sql constraints with the other private attributes, right after _name, _description and _order